src/dialogs/editor/index.tsx

```diff
@@ -120,8 +119,18 @@ window.addEventListener("load", () => {
 	itemBoxLabel.textContent = "Target"; //Wikicite.getString("wikicite.editor.title");
 	container.appendChild(itemBoxLabel);
 	// "item-box" was renamed to "info-box" in Zotero 7.0.10. We compare to 7.0.9 to include the beta versions.
-	const tagName =
-		compareSemVer(Zotero.version, "7.0.9") === 1 ? "info-box" : "item-box";
+	let semVerCompare: number;
+	try {
+		semVerCompare = compareSemVer(Zotero.version, "7.0.10");
+	} catch (e) {
+		// Zotero.version is not a valid semver string
+		// As this may happen in development versions, we will treat it as Zotero 7.0.10
+		Zotero.log(
+			`Zotero version (${Zotero.version}) is not a valid semver string. Will treat it as Zotero 7.0.10.`,
+		);
+		semVerCompare = 1;
+	}
+	const tagName = semVerCompare === 1 ? "info-box" : "item-box";
```

Comment:
Shouldn't this be `>=0` if we want at least 7.0.10?

Reply:
You are absolutely right